src/wallet/wallet.h

```diff
@@ -553,6 +553,12 @@ class CAccountingEntry
     std::vector<char> _ssExtra;
 };
 
+enum CreateTransactionFlags {
+    CREATE_TX_DEFAULT     = 0,
+    CREATE_TX_DONT_SIGN   = (1U << 0),
```

Comment:
Reverted for switch to coin control in 2fce406a978cfc391aa3cc3d66ff4810d48c99e0.